test/game_mock.go

```diff
@@ -16,7 +18,28 @@ var MockGame = models.Game{
 	Active:       true,
 }
 
+var MockUser = models.User{
+	Id:          "abc123",
+	Name:        "test",
+	TotalAmount: 0,
+}
+
+var MockPermission = models.Permission{
+	UserId:      "abc123",
+	Permissions: pq.Int32Array{models.MasterAdmin},
+}
+
 func SeedGame(game *models.Game, db *gorm.DB) error {
 	res := db.Create(game)
 	return res.Error
 }
+
+func SeedUser(user *models.User, db *gorm.DB) error {
```

Comment:
We should keep the mocks in separate `*_mock.go` files to make it easier to keep track of things if we mock a lot of stuff in the future.

move the SeedUser function and the MockUser to `user_mock.go`